tools/compare.py

```diff
@@ -56,6 +56,12 @@ def create_parser():
         help="Do not use colors in the terminal output"
     )
 
+    parser.add_argument(
+        '-d',
```

Comment:
`-j` maybe?

Reply:
I have no strong preference for either. Perhaps my vote would go for `-d` (or `-o`) as many tools are using `-j` when one wants to enable parallelism within them.

Reply:
Right. Let's keep as-is for now i guess.